addons/binding/org.openhab.binding.pioneeravr/src/main/java/org/openhab/binding/pioneeravr/internal/discovery/PioneerAvrDiscoveryParticipant.java

```diff
@@ -118,18 +131,17 @@ public ThingUID getThingUID(RemoteDevice device) {
 
     /**
      * Return true only if the given device model is supported.
-     * 
+     *
      * @param deviceModel
      * @return
      */
-    private boolean isSupportedDeviceModel(final String deviceModel) {
+    private boolean isSupportedDeviceModel(String deviceModel, Set<String> supportedDeviceModels) {
         return StringUtils.isNotBlank(deviceModel)
-                && !Collections2.filter(PioneerAvrBindingConstants.SUPPORTED_DEVICE_MODELS,
-                        new com.google.common.base.Predicate<String>() {
-                            public boolean apply(String input) {
-                                return StringUtils.startsWithIgnoreCase(deviceModel, input);
-                            }
-                        }).isEmpty();
+                && !Collections2.filter(supportedDeviceModels, new Predicate<String>() {
```

Comment:
I know that you didn't write this, but as you have touched it, let me ask: Could this be now done with Java8 streams instead of Google Guava?

Reply:
I am not realy accustomed to either of them as 'Java' is not realy my cup of coffee yet. I am getting used to it while contributing to this project. I am planning to contribute some more for this binding so i could have a look into that in a future PR?

Reply:
No worries, if you do not feel confident enough about such changes, leave that out here!

Reply:
ok.. is there anything i have to do to get this pending "request for change" status removed now? and/or get this back to your attention?

Reply:
The usual way is to add a comment on the main PR saying that you are done with addressing all comments.

Reply:
`&& PioneerAvrBindingConstants.SUPPORTED_DEVICE_MODELS.stream().anyMatch(s -> StringUtils.startsWithIgnoreCase(deviceModel, s))`